src/core/Map/Layers/utils.ts

```diff
@@ -50,12 +50,13 @@ export const copyLazyLayer = (l: LazyLayer | undefined) => {
     if (key === "computed") {
       res = {
         ...res,
-        get [key]() {
-          return computedLayerKeys.reduce((computedRes, computedKey) => {
-            computedRes[computedKey] = l?.[key as keyof LazyLayer]?.[computedKey];
-            return computedRes;
-          }, {} as Record<string, any>);
-        },
+        // get [key]() {
+        //   return computedLayerKeys.reduce((computedRes, computedKey) => {
+        //     computedRes[computedKey] = l?.[key as keyof LazyLayer]?.[computedKey];
+        //     return computedRes;
+        //   }, {} as Record<string, any>);
+        // },
+        [key]: l?.[key], // For debug. We can not access to `computed` on plugin.
```

Comment:
`computed` is cloned(or copy) deeply on quickjs runtime, so it makes application be heavy.